templates/download/risc-v/deepcomputing-fml13v01.html

```diff
@@ -0,0 +1,66 @@
+<div tabindex="0" role="tabpanel" id="deepcomputing-fml13v01-tab" aria-labelledby="deepcomputing-fml13v01"
+  aria-hidden="true">
+  <div class="row p-strip is-shallow u-no-padding--top">
+    <div class="col-3">
+      <h2 class="u-hide--small">Deep<wbr />Computing FML13V01</h2>
```

Comment:
Acts a bit weird in screen reader (similar to hidden br), where it would read "Deep" and "Computing" as separate items in the heading. Not sure it's better than having a space there, but at least closer to the intended branding.

So probably fine to keep as is?